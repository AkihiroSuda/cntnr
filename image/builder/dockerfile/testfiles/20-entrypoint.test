FROM alpine:3.7
ENTRYPOINT ["/entrypoint.sh", "-x"]
ENTRYPOINT /entrypoint.sh -x