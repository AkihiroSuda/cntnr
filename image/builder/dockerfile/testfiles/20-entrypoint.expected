FROM alpine:3.7
NAME 0

ENTRYPOINT "/entrypoint.sh" "-x"
ENTRYPOINT "/bin/sh" "-c" "/entrypoint.sh -x"